src/components/home/setting-modal/helpers.js

```diff
@@ -48,6 +49,36 @@ export const useOptionGroup = (t) => {
           },
         ],
       },
+      {
+        configName: 'documentColor',
+        configLabel: t('documentColor.name'),
+        options: [
+          {
+            value: 'light',
+            label: t('documentColor.light'),
+          },
+          {
+            value: 'dark',
+            label: t('documentColor.dark'),
+          },
+        ],
+      },
     ];
   }, [t]);
 };
+
+export const useDisplayConfig = (initialConfig = asConfigData()) => {
```

Comment:
Why do you define this hook here? I don't think it's a setting-modal-based helper. Instead, it's more like a displayConfig module.